kiwi/filesystem/isofs.py

```diff
@@ -15,7 +15,7 @@
 # You should have received a copy of the GNU General Public License
 # along with kiwi.  If not, see <http://www.gnu.org/licenses/>
 #
-import os
+from distutils.spawn import find_executable
```

Comment:
Hmm, I have checked python-base and python3-base down to SLE11 SP4 and distutils.spawn is part of the core python. In other words from 2.6 to 3.5 it is available. That imho spans a big enough range.